products/public-websites/Cerner-Support/how_to_set_up_vamc_cerner_within_va_health_care_portals.md

```diff
@@ -0,0 +1,46 @@
+# How to set up a VAMC's Cerner integration within the VA.gov health care portals
+
+## Purpose
+
+If you just learned that a VAMC will soon be integrating into Cerner's health care products, then this document will walk you through how to prepare for it.
+
+## Steps
+
+For the following steps, we will use the __Alaska VA Medical Center__ as an example.
+
+1. Obtain the facility ID of the facility that will soon go live as a Cerner facility.
+
+  - In our example, we determine from leadership that the facility ID of the Alaska VA Medical Center is __463__.
+
+1. Create a Pull Request on [`vets-api`](https://github.com/department-of-veterans-affairs/vets-api/pulls) that adds the facility ID as a feature toggle. In our example, the Alaska VA Medical Center has a facility ID of 463, so we would add:
+
+```yml
+cerner_override_463:
+  actor_type: user
+  description: >
+    This will show the Cerner facility 463 as `isCerner`.
```

Comment:
Out of scope but it might be a good pattern to start in the future to use the VAMC's name in the `description` field here like `This will enable Cerner integration for the Alaska VA Medical Center`